tests/unit/core/middleware/test_cspMiddleware.js

```diff
@@ -217,6 +217,18 @@ describe(__filename, () => {
   });
 
   describe('noScriptStyles', () => {
+    // The "noscript styles" have been added for the old
+    // `InstallButton`/`InstallSwitch` components, which have been removed and
+    // replaced with the `AMInstallButton`. We don't use the `noScriptStyles`
+    // feature anymore, but we kept the code because it could still be handy.
+    // See: https://github.com/mozilla/addons-frontend/issues/6540
```

Comment:
I added this link so that next time we need the `noScriptStyles` feature, we come back here and read the log of why this test case was written that way. Otherwise we'll forget and waste time thinking about it 😋